src/entities/Aws.js

```diff
@@ -0,0 +1,8 @@
+export default class Aws {
+  constructor(region, az, instanceType, data) {
```

Comment:
It looks like these 3 properties don't get any special treatment over the rest of the data, and I only mentioned creating this entity because I thought we were going to map the underscored properties in here. Since you found a better way to do it, we probably don't need this entity.

Reply:
++ Agreed.